tests/cupy_tests/test_cudnn.py

```diff
@@ -343,3 +343,115 @@ def test_call(self):
         else:
             with self.assertRaises(self.err):
                 self.call()
+
+
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float16, numpy.float32, numpy.float64],
+    'ksize': [1, 3, 5],
+    'stride': [1, 2, 4],
+    'auto_tune': [True, False],
+    'layout': layouts
+}))
+@unittest.skipUnless(cudnn_enabled, 'cuDNN is not available')
+class TestConvolutionInvalid(unittest.TestCase):
+
+    def setUp(self):
+        self._workspace_size = cudnn.get_max_workspace_size()
+        if libcudnn.getVersion() < 7500:
+            self.skip = True
+            self.skip_msg = "cuDNN version is older than 7.5.0"
+            return
```

Comment:
You can decorate the class with the following instead.
```
@unittest.skipIf(libcudnn.getVersion() < 7500, 'cuDNN version is older than 7.5.0')
```

Reply:
Thanks! I will fix it in that way.